### DISCUSSION TITLE
"unlikely to be verified until you have completed your GitHub user profile with your full name"

### DISCUSSION THREAD

**Discussion by oscie57** [community/community#61537] [GitHub Education]
### Hi everyone,

When trying to apply for GitHub Student Pack, I get this error:

> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

My profile has my full name in both the name, and README. I have no clue what I should do.

**Comment by samus-aran** [comment]
Hi @oscie57 👋 

Welcome! If you have updated your profile with the correct information then perhaps the image submitted can not be read to cross-reference.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Hope this helps 👍

**Comment by MZPL** [comment]
I have the same issue and nothing seems to work. I tried removing and adding my name, bio etc. back which seemed to help, because I managed to submit it successfully for the first time but it got rejected a moment later with the same reason. I really doubt it was even checked by a human. It makes it much harder since I'm forced to use the front camera as selecting the back one doesn't do anything.

**Reply by samus-aran** [reply]
Hi @MZPL 

Here is a quick tip on submitting evidence:

- If your original document is not in English (or includes special characters) then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

**Reply by MZPL** [reply]
@samus-aran Thanks for the suggestion! I printed the translation of my school id including my full name, date of birth, issue date, expiry date and my school name (original and translated), but it didn't work. I came up with an idea to remove diacritics from my name on GH and the print as well and got approved instantly. I don't know if it matters, but this is what I did step by step on my phone:
- cancelled the submission 
- removed full name, bio 
- logged out from GH, GH edu and logged back in
- set my name to my real full name on GH (with accent marks replaced to english letters) and similar bio
- logged out from GH and logged back in
- logged in to GH edu, applied, took photo using my phone's front camera and got approved

Hope someone will find it useful..

**Reply by samus-aran** [reply]
Awesome to hear @MZPL 🎉 

A reminder that when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away if successful. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

It's great to have you in the GitHub Education community and we hope to see you around 👋
